pubspec.yaml

```diff
@@ -12,20 +12,21 @@ dependencies:
   flutter:
     sdk: flutter
 
+  # do not change to cupertino_icons to ^1.0.2 until dart enforces null safety.  This should be next flutter/dart major udpate
   cupertino_icons: ^1.0.0
   provider: ^4.3.2+3
-  get_it: ^5.0.3
+  get_it: ^5.0.4
```

Comment:
Updating existing dependencies is not in the scope of this branch, unless you absolutely require it.

Reply:
Sorry, my add on updated everything...I'll make sure to double check to only update relevant dependencies from now on.